config/initializers/sidekiq.rb

```diff
@@ -1,3 +1,5 @@
 Sidekiq.configure_server do |config|
+  # NOTE: While this may be necessary, this will cause Sidekiq to pound your Redis server
+  # polling for jobs - which is why Mike Perham says that Sidekiq isn't suited for precision timing
```

Comment:
Yeah, I know. But it totally works. It's not like redis isn't able to handle this amount of polling requests.

Reply:
was looking at the WarpWorker and the timing needed to make that work correctly.
perhaps if the front-end sent a notice to the backend that the alignment countdown was done -- and THEN the backend did the warp, then there wouldn't be any perceived issues with "lag".  The notice could include a csrf-token and a "warp signature" to make spoofing very difficult.

since warping is always 10seconds... perhaps removing the countdown would help with perceived "lag" as sidekiq cannot always ensure that things will happen in exactly 10 seconds.